srcpkgs/CBQN/template

```diff
@@ -0,0 +1,50 @@
+# Template file for 'CBQN'
+pkgname=CBQN
+version=0.2.0
+revision=1
+archs="x86_64*"
+_singeli_hash=853ab1a06ae8d8603f228d8e784fa319cc401459
+_replxx_hash=1da4681a8814366ec51e7630b76558e53be0997d
+_bytecode_hash=78ed4102f914eb5fa490d76d4dcd4f8be6e53417
+hostmakedepends="clang libffi-devel"
+depends="libffi"
+short_desc="BQN implementation in C"
+maintainer="Bryce Vandegrift <bryce@brycevandegrift.xyz>"
+license="GPL-3.0-only, MIT, Apache-2.0, BSL-1.0"
+homepage="https://github.com/dzaima/CBQN"
+distfiles="https://github.com/dzaima/CBQN/archive/refs/tags/v${version}.tar.gz
+ https://github.com/mlochbaum/Singeli/archive/${_singeli_hash}.tar.gz
+ https://github.com/dzaima/replxx/archive/${_replxx_hash}.tar.gz
+ https://github.com/dzaima/cbqnBytecode/archive/${_bytecode_hash}.tar.gz"
+checksum="70525117364edefb6c8906e269a5b1b17c8647f516b6599fecdc53ba6e57530e
+ a0a8b48aac23bfa458c3af13cfcad1fb6889151e72b7983e85521245e367c1d4
+ 60ba4d8b566d8af1330f2f4b67d712da8efeb018f53f5055898d8b86a325088f
+ f99c10fc674e673bd8253a90e9676d070aadb4decc3f59766bed3b0295f47a1a"
+
+post_extract() {
+	rm -r CBQN-${version}/build/{bytecode,replxx,singeli}Submodule
+	mv "cbqnBytecode-${_bytecode_hash}" "CBQN-${version}/build/bytecodeSubmodule"
+	mv "replxx-${_replxx_hash}" "CBQN-${version}/build/replxxSubmodule"
+	mv "Singeli-${_singeli_hash}" "CBQN-${version}/build/singeliSubmodule"
+	cd CBQN-${version}
+	vsed -i makefile -e 's/@git/@true/g'
+	vsed -i makefile -e '/SHELL =.*/ d'
+}
+
+do_build() {
+	cd CBQN-${version}
+	make REPLXX=1 j=$(nproc)
```

Comment:
could set `make_build_args="REPLXX=1"` and just use `build_style=gnu-makefile`

Reply:
I put this here because the build system cannot find the makefile so I have to manually `cd` into `CBQN-${version}`.

Reply:
Yeah, it'll switch to the right directory if you set `build_wrksrc` as mentioned before.